help/destinations/catalog/crm/zendesk.md

```diff
@@ -112,21 +112,21 @@ To correctly map your XDM fields to the [!DNL Zendesk] destination fields, follo
 
 1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
 1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
-1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select attributes]** category and select from the displayed schema matching a desired attribute name from the list of [[!DNL Zendesk] Contacts API fields](https://developer.zendesk.com/api-reference/sales-crm/resources/contacts/).
```

Comment:
Looking at the mapping in the UI, the target schema is already configured for Zendesk. So what is the point of directing readers to the Zendesk API docs? They can just select one of the attributes, they don't need to validate that they match what Zendesk has. They are already validated.

Let's rephrase to:

In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** category and select a target identity, or choose the **[!UICONTROL Select attributes]** category and select one of the supported schema attributes.